### ISSUE TITLE
reduce argument footprint for marbl_compute_PAR

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#31]
Currently passing `marbl_mod::marbl_compute_PAR` entire `autotroph_local_type` array when we just need the `Chl` attribute of it.

**Comment by mnlevy1981** [comment]
Want

```
real (r8) :: totalChl_local(domain%km)   ! local value of totalChl
```

in `marbl_set_interior_forcing()`, and it should be set in `marbl_setup_local_tracers()`

**Comment by mnlevy1981** [comment]
closed via #182
